nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -2093,6 +2093,12 @@
       change_function:
         '_overrides': "'(lambda method, *args, **kwargs: method)'"
 
+- module-name: 'qtawesome'
+  data-files:
+    dest_path: 'qtawesome'
```

Comment:
This looks wrong or unecessary, I think you should remove it.

Reply:
the dest_path?
let me have a test